go.sum

```diff
@@ -1,10 +1,19 @@
+<<<<<<< Updated upstream
```

Comment:
unresolved stash pop conflict i think?

Reply:
yes. I don't know how this got here. thought go mod tidy would complain if I had weird things in here. I'll fix it.